src/webextensionadapter.cpp

```diff
@@ -4,6 +4,9 @@
 
 #include "webextensionadapter.h"
 
+#include <qjsonarray.h>
+#include <qjsonobject.h>

```

Comment:
Aren't these already included on lines 11 and 13?

Reply:
truee, i have a new vscode addon that probably added that >:(